proto/stride/stakeibc/tx.proto

```diff
@@ -131,10 +131,10 @@ message MsgRegisterHostZoneResponse {}
 
 message MsgClaimUndelegatedTokens {
   string creator = 1;
-  // UserUnbondingRecords are keyed on {chain_id}.{epoch}.{sender}
+  // UserUnbondingRecords are keyed on {chain_id}.{epoch}.{receiver}
   string host_zone_id = 2;
   uint64 epoch = 3;
-  string sender = 4;
+  string receiver = 4;
```

Comment:
Why use a new field / reserve the old one above in UserRedemptionRecord, but not here?

Reply:
I didn't think it mattered for messages since we don't read them from state. Probably low cost to use reserve though

Reply:
updated